src/nvim/version.c

```diff
@@ -1067,31 +1067,6 @@ void list_version(void)
   version_msg(SYS_VIMRC_FILE);
   version_msg("\"\n");
 #endif  // ifdef SYS_VIMRC_FILE
-#ifdef USR_VIMRC_FILE
-  version_msg(_("     user vimrc file: \""));
-  version_msg(USR_VIMRC_FILE);
-  version_msg("\"\n");
-#endif  // ifdef USR_VIMRC_FILE
-#ifdef USR_VIMRC_FILE2
-  version_msg(_(" 2nd user vimrc file: \""));
-  version_msg(USR_VIMRC_FILE2);
-  version_msg("\"\n");
-#endif  // ifdef USR_VIMRC_FILE2
-#ifdef USR_VIMRC_FILE3
-  version_msg(_(" 3rd user vimrc file: \""));
-  version_msg(USR_VIMRC_FILE3);
-  version_msg("\"\n");
-#endif  // ifdef USR_VIMRC_FILE3
-#ifdef USR_EXRC_FILE
-  version_msg(_("      user exrc file: \""));
-  version_msg(USR_EXRC_FILE);
-  version_msg("\"\n");
-#endif  // ifdef USR_EXRC_FILE
-#ifdef USR_EXRC_FILE2
-  version_msg(_("  2nd user exrc file: \""));
-  version_msg(USR_EXRC_FILE2);
-  version_msg("\"\n");
-#endif  // ifdef USR_EXRC_FILE2
```

Comment:
This breaks the purpose of `:version`. And `$MYVIMRC` is still there. Also vimrc file name placed here means that without `-u` Neovim will *always* source *exactly this* vimrc which

1. Is useless if it is written as `$XDG_CONFIG_HOME/.config/init.vim`. (BTW, I have not yet implemented #78/1/ii which will make this variant look even more strange.)
2. Contains even more information which has nothing to do with compilation options if I also provide $XDG_CONFIG_HOME (and $XDG_CONFIG_DIRS) values in addition to their defaults.
3. Is generally incorrect if I perform the expansion before providing any information, and also will contain lots of unrelated information with 1/ii.
4. Is useless for the purpose of determining “where I need to put init.vim file” in case I provide only the effective file from all the variants with 1/ii (still useful if 1/ii is not implemented).